src/qibolab/tests/test_instruments_qmsim.py

```diff
@@ -10,10 +10,9 @@
 from qibo.models import Circuit
 
 from qibolab.backends import QibolabBackend
-from qibolab.instruments.qm import QMOPX, QMPulse, QMSequence
 from qibolab.paths import qibolab_folder
 from qibolab.platform import create_tii_qw5q_gold
-from qibolab.pulses import FluxPulse, Pulse, PulseSequence, ReadoutPulse, Rectangular
+from qibolab.pulses import FluxPulse, PulseSequence, Rectangular
 
 REGRESSION_FOLDER = pathlib.Path(__file__).with_name("regressions")
```

Comment:
Instead of using a folder in the repo, I would just use a `/tmp` subfolder, downloading there existing data and writing there the new results.
We could control this with an env var.

Reply:
In any case, this definition can be moved in `conftest.py`.